src/dispatch/feedback/service/models.py

```diff
@@ -21,14 +20,17 @@ class ServiceFeedback(TimeStampMixin, FeedbackMixin, Base):
     feedback = Column(String)
     rating = Column(String)
     schedule = Column(String)
-    hours = Column(Integer)
+    hours = Column(Numeric(precision=10, scale=2))
     shift_start_at = Column(DateTime)
     shift_end_at = Column(DateTime)
 
     # Relationships
     individual_contact_id = Column(Integer, ForeignKey("individual_contact.id"))
```

Comment:
If the individual were to get deleted, do you want to keep the feedback or delete it? If the latter, then you want to add `ondelete="CASCADE"`

`individual_id = Column(Integer, ForeignKey("individual_contact.id", ondelete="CASCADE"))`

Reply:
I think we should keep the feedback even if the individual is deleted.